people.go

```diff
@@ -23,11 +23,12 @@ import (
 	"github.com/src-d/identity-matching/reporter"
 )
 
-// rawPerson is taken from a single commit signature with only one name and one email
-type rawPerson struct {
+// Commit is taken from a single commit signature with only one name and one email
+type Commit struct {
 	repo  string
 	name  string
 	email string
```

Comment:
```suggestion
	object.Signature
```

https://github.com/src-d/go-git/blob/8e76874ae2a3f5029269af76a86f0ee294699df9/plumbing/object/object.go#L81

```go
type signatureWithRepo struct {
    object.Signature
    Repository string
}
```

Reply:
So I should import go-git then, or just use the same logic?

Reply:
Nah I forgot that we don't use go-git here. It makes no sense to depend on it just for using a single trivial type